akka-diagnostics/src/main/scala/akka/diagnostics/ConfigChecker.scala

```diff
@@ -805,10 +843,12 @@ class ConfigChecker(system: ExtendedActorSystem, config: Config, reference: Conf
   private def checkCluster(): Vector[ConfigWarning] =
     if (isClusterConfigAvailable) {
       Vector.empty[ConfigWarning] ++
-      checkAutoDown() ++
+      checkAutoDown() ++ //TODO move isClusterConfigAvailable to each of the methods below?
       checkClusterFailureDetector() ++
-      checkClusterDispatcher()
-    } else Vector.empty[ConfigWarning]
+      checkClusterDispatcher() ++
+      checkCreateActorRemotely()
+//      checkClusterWatchFailureDetector() //TODO if cluster depends on remote this will warn everytime we use `akka.actor.provider = cluster`?
```

Comment:
I meant that I don't understand what should not be allowed in `akka.remote.watch-failure-detector` if it comes as a dependency from Akka-cluster. By using `akka-cluster` it brings the `akka-remote` `reference.conf` with it and in turn all the `akka.remote.watch-failure-detector` values. Obviously, I'm missing something here.

Reply:
I'm referring here to implementing `remote watch failure detector (shouldn't be used when cluster is used)` from #13

Reply:
`akka.remote.watch-failure-detector` comes from the `akka-remote` module not `akka-cluster`.
These new checks should not be part of the checkCluster but instead included in checkRemote

Reply:
Yep :) I was aware it came from `akka-remote` as pointed out in the [comment](https://github.com/akka/akka-diagnostics/pull/21/files#r1069354522). I thought the requirement was that has to be checked when the `akka.actor.provider=cluster` because of the description in #13 `(shouldn't be used when cluster is used)`. Hence my question in the [comment](https://github.com/akka/akka-diagnostics/pull/21/files#r1069354522). 
In any case you clarified this question with your comment above and [this other one](https://github.com/akka/akka-diagnostics/pull/21/files#r1069582939).